.github/workflows/build-and-publish-release-artifacts.yml

```diff
@@ -0,0 +1,57 @@
+name: Build and Publish Release Artifacts
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+
+  build-and-publish:
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: ^1.17.0
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v3.0.2
+
+    - run: mkdir roadmap-graphviz-linux-amd64 && env GOOS=linux GOARCH=amd64 go build -o roadmap-graphviz-linux-amd64 ./... && zip -r roadmap-graphviz-linux-amd64.zip roadmap-graphviz-linux-amd64
+    - run: mkdir roadmap-graphviz-windows-amd64 && env GOOS=windows GOARCH=amd64 go build -o roadmap-graphviz-windows-amd64 ./... && zip -r roadmap-graphviz-windows-amd64.zip roadmap-graphviz-windows-amd64
+
+    - name: Test
```

Comment:
minor nit: I'd usually look to run the test suite before building release artifacts.